### PR TITLE
formulaire 2 tuto + code complet

### PR DESCRIPTION
**Pull request by GtAntoine**

### CODE REVIEW

File: src/views/user/card.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const cardStyle= {
+  flex: 1,
+  minWidth: '300px',
+  maxWidth: '300px',
+  marginTop: '7px',
+  marginRight: '20px',
+  marginBottom: '20px'
+};
+
+const Card = ({title,description, route, destination }) => {
```
Comment (TomGallon) [inline src/views/user/card.js]: Please remove this component.

**Review by TomGallon — CHANGES_REQUESTED** [review]

File: src/config/translations/fr-FR.js [hunk 2]
```diff
@@ -32,5 +32,9 @@ export default {
         male: 'Homme',
         female: 'Femme',
         unknown: 'Inconnu'
+    },
```
Comment (Ephrame) [inline src/config/translations/fr-FR.js]: it's a good i18n key, you have to indicate the name of the form. It's specific to one form.

File: src/services/index.js [hunk 3]
```diff
@@ -1 +1,16 @@
-// déclarez ici les appels aux API de services métier
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadFinance = async ({id}) => {
+    const response = await focusFetch({url: `http://localhost:9999/x/complex/${id}`, method: 'get'})
+    const data = await response;
+    return { ...data.finance, __Focus__updateRequestStatus: data.__Focus__updateRequestStatus };
+}
```
Comment (Ephrame) [inline src/services/index.js]: You have to make a specific webService

File: src/views/02-Les listes/README.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+Ajoutez les nouveaux services
+
+```jsx
+// services/index.js
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadFinance = async ({id}) => {
+    const response = await focusFetch({url: `http://localhost:9999/x/complex/${id}`, method: 'get'})
+    const data = await response;
+    return { ...data.user, __Focus__updateRequestStatus: data.__Focus__updateRequestStatus };
```
Comment (Ephrame) [inline src/views/02-Les listes/README.md]: You want data.finance and not user

File: src/views/02-Les listes/README.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+}
+
+export const saveFinance = async ({user}) => {
+    await new Promise((resolve, reject) => {
```
Comment (Ephrame) [inline src/views/02-Les listes/README.md]: it's finance not user

File: src/views/02-Les listes/README.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+export default FinancialMoveLine;
+```
+
+N'oubliez pas de rajouter la route dans home.js...
+
+```jsx
+{route: '/finance/120', destination: 'Finance List', description: 'Exemple d\'un formulaire avec un ListFor', title: 'Finance List'}
+```
+
+...et d'importer la vu dans le router !
```
Comment (Ephrame) [inline src/views/02-Les listes/README.md]: vue

File: src/views/02-Les listes/README.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+import {connect as connectToFieldHelpers} from 'focus-graph/behaviours/field';
+import {loadFinanceAction, saveFinanceAction} from '../../actions/finance-actions';
+
+import Panel from 'focus-components/panel';
+import {compose} from 'redux';
+import FinancialMoveLine from './financialMoveLine'
+
+const Finance = ({fieldFor,listFor, ...otherProps}) => (
+    <Panel title='Finance ' {...otherProps}>
+        {fieldFor('name', {entityPath: 'finance'})}
```
Comment (Ephrame) [inline src/views/02-Les listes/README.md]: The entityPath is not necessary

File: src/views/02-Les listes/README.md [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+Ajoutez les nouveaux services
+
+```jsx
+// services/index.js
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadFinance = async ({id}) => {
+    return focusFetch({url: `http://localhost:9999/x/finances/${id}`, method: 'GET'}).then((data) => {
+        return {
+            ...data.finance,
```
Comment (TomGallon) [inline src/views/02-Les listes/README.md:122]: please remove the THEN.

File: src/views/home.js [hunk 9]
```diff
@@ -1,9 +1,27 @@
...
-    render() {
-        return <h1>Bienvenue sur le tutoriel de Focus v3</h1>
-    }
+const routes = [
+    {route: '/finances/120', destination: 'Finance List', description: 'Exemple d\'un formulaire avec un ListFor', title: 'Finance List'}
+];
+
+const Home = props => {
+    return(
+        <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center'}}>
```
Comment (TomGallon) [inline src/views/home.js]: Could you please avoid to use style property ?
We recommand to declare CSS in project instead.

**Review by TomGallon — CHANGES_REQUESTED** [review]

File: src/config/translations/fr-FR.js (hunk 2 above)
Comment (GtAntoine) [reply src/config/translations/fr-FR.js]: This remark is not clear for me

**Pull request merged**